rust/description.md

```diff
@@ -0,0 +1,15 @@
+## Rust
+
+Two different implementations are provided for Rust:
+
+ - `revcmp_array` is a direct adoption of the revcomp C variant
+ - `revcomp_iter` is an idiomatic variant that uses iterators to iterate the array in reverse order, translate and then collect into the result vector.
+
+&&&& src/lib.rs
+
+Interestingly, the idiomatic variant using the iterator is significantly faster than the array-variant. This also 
+holds when benchmarked against an unsafe variant of the array version that omits array-bound checks. Run
+`cargo bench` run the benchmarks. 
```

Comment:
If the idiomatic version is better then we don't need the other one.

Reply:
Let's not reference cargo on the website.

Reply:
Ok - as written above, I am fine with keeping the iterator variant :)